docs/src/model_creation/network_analysis.md

```diff
@@ -228,11 +224,10 @@ rn = @reaction_network begin
 end
 ```
 with graph
-```julia
-complexgraph(rn)
+```@example s1
+plot_complexes(rn)
```

Comment:
Similar comment about dropping edge labels, and about the huge amount of white space.